frontend/components/utils/AnimatedNumber.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <animated-number
         v-if="value || value === 0"
```

Comment:
Since value can be a big number 0, isn't it better to check it against undefined and null, same as in the other file?

Reply:
I added the same check on this component.